src/System.Text.Json.Dynamic/System/Text/Json/JsonDynamicObject.cs

```diff
@@ -10,7 +10,17 @@ namespace System.Text.Json
 {
     public class JsonDynamicObject : DynamicObject, IBufferFormattable
     {
-        Dictionary<JsonProperty, JsonValue> _properties;
+        #region Constants
+
+        private static readonly char[] EncodingChars = new char[] { '{', '}', ',', ':', '"' };
```

Comment:
I added that because of the other usage initially then realized I was in 2 separate classes and forgot to remove it. I'll strip it out.